### PR TITLE
Add Reverse audio

### PR DESCRIPTION
**Pull request by atamazian**
Adds new augmentation called `Reverse`, which allows to reverse audio. Implements https://github.com/iver56/audiomentations/issues/117.

### CODE REVIEW

File: tests/test_reverse.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import unittest
+
+import numpy as np
+
+from audiomentations import Reverse
```
Comment (iver56) [inline tests/test_reverse.py:5]: When running `pytest`, I get the following error:

```______________________________________ ERROR collecting tests/test_reverse.py ______________________________________
ImportError while importing test module 'C:\Users\Iver\Code\audiomentations\tests\test_reverse.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests\test_reverse.py:5: in <module>
    from audiomentations import Reverse
E   ImportError: cannot import name 'Reverse' from 'audiomentations' (C:\Users\Iver\Code\audiomentations\audiomentations\__init__.py)
```

To fix this error, either change the import here or add `Reverse` to the list of publicly exposed transforms in audiomentations/\_\_init\_\_.py

File: tests/test_reverse.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    def test_multichannel(self):
+        samples = np.array(
+            [[0.9, 0.5, -0.25, -0.125, 0.0], [0.95, 0.5, -0.25, -0.125, 0.0]],
+            dtype=np.float32,
+        )
+        sample_rate = 16000
+        augmenter = Reverse(p=1.0)
+        samples = augmenter(samples=samples, sample_rate=sample_rate)
+
+        self.assertEqual(samples.dtype, np.float32)
```
Comment (iver56) [inline tests/test_reverse.py:27]: Please add an assertion that checks that the output is reversed as expected when applied to multichannel audio:

```
        assert_array_almost_equal(
            reversed_samples,
            np.array(
                [[0.0, -0.125, -0.25, -0.5, 0.9], [0.0, -0.125, -0.25, 0.5, 0.95]],
                dtype=np.float32,
            ),
        )
```

You may find that the test fails after you add this assertion. In that case, the implementation needs to be fixed. Maybe slice notation is better than np.flipud?

**Review by iver56 — CHANGES_REQUESTED** [review]
Thanks for the initiative :)

File: tests/test_reverse.py (hunk 1 above)
Comment (atamazian) [reply tests/test_reverse.py:5]: Done.

File: tests/test_reverse.py (hunk 2 above)
Comment (atamazian) [reply tests/test_reverse.py:27]: Can you tell me what is the source of `assert_array_almost_equal` function?

Comment (iver56) [reply tests/test_reverse.py:27]: Sorry, I forgot to mention that. It's `from numpy.testing import assert_array_almost_equal`

Comment (atamazian) [reply tests/test_reverse.py:28]: I've fixed that problem and updated test file, please check.

**Review by iver56 — APPROVED** [review]

**Comment by iver56** [comment]
Thanks!

**Pull request merged**